apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx

```diff
@@ -110,7 +110,7 @@ const UtilityTabResults = ({ id, isExecuting }: UtilityTabResultsProps) => {
                   </pre>
                 ))
               ) : (
-                <p className="font-mono text-sm">{result.error.error}</p>
+                <p className="font-mono text-sm">{result.error.error?.message}</p>
```

Comment:
Recently, if we type something wrong in a query, instead of showing an error, it throws an exception because our response from pg-meta has changed; it is now an object.